stable/datadog/values.yaml

```diff
@@ -1,290 +1,256 @@
   ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
-  ##
-  # nonLocalTraffic: true
+  #
+  #  nonLocalTraffic: true
 
+  ## @param dogstatsdOriginDetection - boolean - optional
   ## Enable origin detection for container tagging
   ## https://docs.datadoghq.com/developers/dogstatsd/unix_socket/#using-origin-detection-for-container-tagging
-  ##
-  # dogstatsdOriginDetection: true
+  #
+  #  dogstatsdOriginDetection: true
 
+  ## @param useCriSocketVolume - boolean - required
   ## Enable container runtime socket volume mounting
+  #
   useCriSocketVolume: true
 
+  ## @param useDogStatsDSocketVolume - boolean - optional
   ## Enable dogstatsd over Unix Domain Socket
   ## ref: https://docs.datadoghq.com/developers/dogstatsd/unix_socket/
-  ##
-  # useDogStatsDSocketVolume: true
+  #
+  #  useDogStatsDSocketVolume: true
 
-  ## Set host tags.
-  ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
+  ## @param tags  - list of key:value elements - optional
+  ## List of tags to attach to every metric, event and service check collected by this Agent.
   ##
-  # tags:
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
 
-  ## Enables event collection from the kubernetes API
+  ## @param collectEvents - boolean - optional - default: false
+  ## Enables this to start event collection from the kubernetes API
   ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
-  ##
-  collectEvents: false
+  #
+  #  collectEvents: false
+
+  ## @param leaderElection - boolean - optional - default: false
+  ## Enables leader election mechanism for event collection.
+  #
+  #  leaderElection: false
 
-  ## Enables log collection
+  ## @param leaderLeaseDuration - integer - optional - default: 600
+  ## Set the lease time for leader election in second.
+  #
+  #  leaderLeaseDuration: 600
+
+  ## @param logsEnabled - boolean - optional - default: false
+  ## Enables this to activate Datadog Agent log collection.
   ## ref: https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#log-collection-setup
-  ##
-  # logsEnabled: false
-  # logsConfigContainerCollectAll: false
+  #
+  #  logsEnabled: false
 
-  ## Un-comment this to enable APM and tracing, on port 8126
+  ## @param logsConfigContainerCollectAll - boolean - optional - default: false
+  ## Enable this to allow log collection for all containers.
+  ## ref: https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#log-collection-setup
+  #
+  #  logsConfigContainerCollectAll: false
+
+  ## @param apmEnabled - boolean - optional - default: false
+  ## Enable this to enable APM and tracing, on port 8126
   ## ref: https://github.com/DataDog/docker-dd-agent#tracing-from-the-host
-  ##
-  # apmEnabled: true
+  #
+  #  apmEnabled: false
```

Comment:
This change is not consistent with dsd's `#  nonLocalTraffic: true`, which style should we keep?